libexec/rtld-elf/rtld_c18n_policy.txt

```diff
@@ -54,6 +54,7 @@ trust
 	execvp
 	execvpe
 	execvP
+	unw_getcontext
```

Comment:
Does `unw_getcontext_unsealed` need to be in this list too?

Reply:
It does, good spot. I'll also add a new libunwind compartment that these symbols get exposed to, much like you have a [TCB] one.